### ISSUE TITLE
[Legitimate Site Blocked] metamask.market

### ISSUE THREAD

**Issue by rukubfab** [MetaMask/eth-phishing-detect#5849]
http://metamask.market/

Hi! I've just registerd this domain, but can't access it, browser telling me its unsafe, but its NEW domain and its mine ))
We don't want to compromise MetaMask, more, we'll give link and instruction to their site, because we'll building NFT Marketplace with metamask authorization in its center. How can we get out from this block? Thanks.

**Comment by devenblake** [comment]
The fuzzy detection happening here is that you registered `metamask.[TLD]` and `metamask.io` is a legitimate website, and users of `metamask.io` are frequently targeted in phishing attacks.

`metamask.market` is rightfully blocked because this appears to be an official MetaMask site and is not. Unfortunately, MetaMask is a [registered trademark](https://tmsearch.uspto.gov/bin/showfield?f=doc&state=4806:gpr2e2.2.3) of ConsenSys Software Incorporated, and a quick skim of 15 U.S.C. §1125 indicates that naming your site "MetaMask Market" or some form of it can make you liable for civil action by ConsenSys.

Basically, your naming your domain `metamask` could have legal consequences, and anyway for the context of this repository it's still too big of a risk (in my opinion without authority) to let `metamask.market` into the allowlist, on the off-chance it becomes a scam later. You should probably just rename your service.
